docs/netdata-agent/versions-and-platforms.md

```diff
@@ -67,3 +67,49 @@ Static builds usually miss certain features that require operating-system suppor
 - eBPF related features
 
 When platforms are removed from the [Binary Distribution Packages](/packaging/makeself/README.md) list, they default to install or update Netdata to a static build. This may mean that after platforms become EOL, Netdata on them may lose some of its features. We recommend upgrading the operating system before it becomes EOL, to continue using all the features of Netdata.
+
+### Upgrading on an Unsupported Platform
+
+Existing installs of native packages on a platform that is no longer supported, cannot be automatically updated at this time. The upgrader will show it is already at the newest version, because we no longer publish new packages.
+
+If the operating system cannot be upgraded to a more recent version, the install can be switched to a static build. This is a manual process, may cause data loss, and is therefore **not supported**. However, following these steps should result in a functioning Agent with metrics data and connection to Netdata Cloud in tact:
+
+1. Stop the Agent, as appropriate for your platform. For platforms with systemd:
+
+   ```sh
+   sudo systemctl stop netdata
+   ```
+
+2. Backup the contents of these directories: `/etc/netdata`, `/var/cache/netdata`, `/var/lib/netdata`, `/var/log/netdata`.
+3. Uninstall the native package, answering "yes" to all questions.
+
+   ```sh
+   wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh --uninstall
+   ```
+4. Unmask the Netdata service for systemd:
+
+   ```sh
+   sudo systemctl unmask netdata
+   sudo systemctl daemon-reload
+   ```
```

Comment:
Here we are again giving info about one specific tool for controlling the Netdata service, I believe this was something we were trying to avoid

cc @ilyam8

Reply:
Well, yes, but read the comment on the ticket to @Ferroin. Not doing this explicitly breaks the install afterwards. I.e. the service file will not work properly.

Reply:
So that service is a systemd one, what happens if the system is not a systemd one? Maybe I am getting something wrong here

Reply:
I have not tested the procedure on a non-systemd install. Don't all native packages use systemd? Maybe except Amazon Linux 2.

The issue here is a workaround for a problem that we created when we added the masking in `postrm`. I don't know why we did this, I don't think it is proper, but until we change that, leaving off these instructions will simply break the install.